include/exadg/convection_diffusion/user_interface/boundary_descriptor.h

```diff
@@ -32,6 +32,8 @@
 // ExaDG
 #include <exadg/functions_and_boundary_conditions/function_cached.h>
 
+#include <memory>
```

Comment:
for example here: Why is memory needed here? We should make sure to insert the includes where necessary.

Reply:
I guess due to the shared pointers.

Reply:
@peterrum exactly